src/main/java/com/palantir/gradle/gitversion/VersionDetails.java

```diff
@@ -141,4 +125,19 @@ public String getBranchName() throws IOException {
 
         return ref.getName().substring(Constants.R_HEADS.length());
     }
+
+    @Override
+    public String toString() {
```

Comment:
This object used to have nice representation when it was written in Groovy. Trying to match that representation with this toString